plugins/woocommerce/client/blocks/tests/e2e/tests/checkout/order-confirmation.block_theme.spec.ts

```diff
@@ -46,6 +46,7 @@ test.describe( 'Shopper (logged-in) → Order Confirmation', () => {
 		await frontendUtils.goToShop();
 		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
 		await frontendUtils.addToCart( SIMPLE_VIRTUAL_PRODUCT_NAME );
+		await frontendUtils.goToCart();
```

Comment:
Is this fixing anything or are we doing it as an extra measure? It will add to the length of a test without any assertions to be checked?

Reply:
I think this can me removed now